mmv1/products/netapp/backup.yaml

```
@@ -0,0 +1,145 @@
+  inside the backup vault.
+
+  You can create manual and scheduled backups. Manual backups can be taken
+  from a volume or from an existing volume snapshot. Scheduled backups
+  require a backup policy.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Documentation': 'https://cloud.google.com/netapp/volumes/docs/protect-data/about-volume-backups'
+  api: 'https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.backupVaults.backups'
+base_url: projects/{{project}}/locations/{{location}}/backupVaults/{{vault_name}}/backups
+self_link: projects/{{project}}/locations/{{location}}/backupVaults/{{vault_name}}/backups/{{name}}
+create_url: projects/{{project}}/locations/{{location}}/backupVaults/{{vault_name}}/backups?backupId={{name}}
+create_verb: :POST
+update_url: projects/{{project}}/locations/{{location}}/backupVaults/{{vault_name}}/backups/{{name}}
+update_verb: :PATCH
+update_mask: true
+delete_url: projects/{{project}}/locations/{{location}}/backupVaults/{{vault_name}}/backups/{{name}}
+delete_verb: :DELETE
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+id_format: 'projects/{{project}}/locations/{{location}}/backupVaults/{{vault_name}}/backups/{{name}}'
+import_format: ['projects/{{project}}/locations/{{location}}/backupVaults/{{vault_name}}/backups/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'netapp_backup'
+    primary_resource_id: 'test_backup'
+    vars:
+      pool_name: 'backup-pool'
+      volume_name: 'backup-volume'
+      backup_vault_name: 'backup-vault'
+      backup_name: 'test-backup'
+    test_vars_overrides:
+      network_name: 'acctest.BootstrapSharedServiceNetworkingConnection(t, "gcnv-network-config-1", acctest.ServiceNetworkWithParentService("netapp.servicenetworking.goog"))'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Location of the backup.
+  - !ruby/object:Api::Type::String
+    name: 'vault_name'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Name of the backup vault to store the backup in.
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description: |
+      The resource name of the backup. Needs to be unique per location.
+    required: true
+    immutable: true
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    description: |
+      The state of the Backup Vault.
+    values:
+      - :STATE_UNSPECIFIED
+      - :CREATING
+      - :UPLOADING
+      - :READY
+      - :DELETING
+      - :ERROR
+      - :UPDATING
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      An optional description of this resource.
```

Api description has some potentially useful additional information?

```suggestion
      A description of the backup with 2048 characters or less. Requests with longer descriptions will be rejected.
```

---

Done